R/pagination.R

```diff
@@ -1161,16 +1161,21 @@ paginate_to_mpfs <- function(obj,
   ## but we will if we ever allow force_paginate to do horiz
   ## pagination.
   if (length(fpags) > 1) {
-    deep_pag <- lapply(
-      fpags, paginate_to_mpfs,
```

Comment:
this was the old list pagination which does not work here. We use directly `paginate_mpfs`